test-common/src/main/kotlin/io/opentelemetry/android/test/common/AssertionExtensions.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.test.common
+
+import io.opentelemetry.sdk.logs.data.internal.ExtendedLogRecordData
+import io.opentelemetry.sdk.testing.assertj.LogRecordDataAssert
+import io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.assertThat
+
+fun LogRecordDataAssert.hasEventName(eventName: String): LogRecordDataAssert {
+    isNotNull()
+    // TODO: This will be removed after the event bits of logs gets out of incubating.
```

Comment:
just a note I guess? or should we create an issue?

Reply:
I think the note is fine....once incubator changes it should force us to notice.